catalogue/webapp/components/WorkDetails/WorkDetails.js

```diff
@@ -29,6 +29,27 @@ import { clientSideSearchParams } from '@weco/common/services/catalogue/search-p
 import Download from '../Download/Download';
 import PaletteSimilarityBox from '../PaletteSimilarityBox/PaletteSimilarityBox';
 
+type TimeProps = {|
+  hours: string,
+  minutes: string,
+  seconds: string,
+|};
+
+function doubleDigits(n) {
+  return n > 9 ? `${n}` : `0${n}`;
+}
+
+function convertMS(milliseconds: number): TimeProps {
+  const seconds = milliseconds / 1000;
+  const closestSeconds = Math.round(seconds);
+  const minutes = Math.floor(closestSeconds / 60);
+  const hours = Math.floor(minutes / 60);
+  return {
+    hours: doubleDigits(hours),
+    minutes: doubleDigits(minutes % 60),
+    seconds: doubleDigits(closestSeconds % 60),
+  };
+}
```

Comment:
Could we not use `moment.duration` for readability?

Reply:
moment.duration doesn't provide the formatting we need, but we can just use moment.utc and format that, which will work for anything up to 24hrs in duration